### PR TITLE
Change edge checker method

### PR DESCRIPTION
**Pull request by Tzugimaa**
Should be better than the last edge checker.
If not, please tell me.

### CODE REVIEW

File: Effects/LCAS.hlsl [hunk 1]
```diff
@@ -58,10 +57,10 @@ float4 Pass1(float2 pos) {
 	//    w  
 	//  w 1 w
 	//    w   
-	// If is not edge
-	if ((0.72f * sum) <= 1)
-		return float4(((((b + d) + (f + h)) * wRGB + e) / (1.0 + 4.0 * wRGB)).rgb, 1);
-	else
-		return float4(((((b + d) + (f + h)) * wRGB + e * (1 + sharpness) * 0.72f * sum) / (1.0 + 4.0 * wRGB)).rgb, 1);
 	// If is edge
+	if(edge >= 0.2)
```
Comment (Blinue) [inline Effects/LCAS.hlsl:61]: How about letting the user decide the threshold?

Comment (Tzugimaa) [reply Effects/LCAS.hlsl:61]: Thats a good idea sir

**Comment by Tzugimaa** [comment]
threshold of 0 almost everything is edge should not be used unless user likes the result.
threshold of 0.1 gets every  edge.
threshold of 0.2 gets almost every edge.
threshold of 1 is likely to get only texts and clear edge.

**Comment by Blinue** [comment]
LGTM

**Pull request merged**